cdec/installation-manager-cli-bundle/src/main/java/com/codenvy/im/cli/command/AuditCommand.java

```diff
@@ -39,11 +37,11 @@
  */
 @Command(scope = "codenvy", name = "audit", description = "Download Audit report and print it on the screen")
 public class AuditCommand extends AbstractIMCommand {
-    private final File auditDirectory;
+    private final Path auditDirectory;
 
     public AuditCommand() {
         auditDirectory =
-                new File(InjectorBootstrap.INJECTOR.getInstance(Key.get(String.class, Names.named("installation-manager.audit_dir"))));
+                Paths.get(InjectorBootstrap.INJECTOR.getInstance(Key.get(String.class, Names.named("installation-manager.audit_dir"))));
```

Comment:
Form the performance and readability point it, better to init path value in constructor.